src/gba/GBA.cpp

```diff
@@ -1712,11 +1712,21 @@ const char* GetSaveDotCodeFile()
     return saveDotCodeFile;
 }
 
+void ResetLoadDotCodeFile()
+{
+    loadDotCodeFile = strdup("");
+}
+
 void SetLoadDotCodeFile(const char* szFile)
 {
     loadDotCodeFile = strdup(szFile);
 }
 
+void ResetSaveDotCodeFile()
+{
+    saveDotCodeFile = strdup("");
```

Comment:
Likewise.

Reply:
I have also fixed this, so please check it.